guides/v2.1/frontend-dev-guide/translations/theme_dictionary.md

```diff
@@ -37,30 +36,34 @@ For example, this approach is used in the Magento Luma theme. It has the [`<Mage
 
 It is important to remember that if you generate a dictionary for your theme using the [i18n tool] with the conventional names and locations for the dictionary, the existing dictionary gets overwritten. 
 
-That is why the recommended flow for adding custom strings is the following:
+We recommend the following workflow to add custom strings:
```

Comment:
Why not a simple title? If we put it there, we must be recommending you do it, especially with the preceding paragraph.
`#### To add custom strings:`

Reply:
Dig it, changing